src/Annotations/Command/QueueWatchCommand.php

```diff
@@ -51,7 +51,13 @@ protected function process(InputInterface $input, QueueItem $item, $entity = nul
                 $this->logger->info(sprintf('No email address for profile "%s", backup email address created.', $id));
                 $email = $id.'@hypothesis.elifesciences.org';
             }
-            $user = new User($id, $email, $display_name);
```

Comment:
will keep an eye on this command, if it grows too much we'll extract the logic from here